### ISSUE TITLE
auto-sklearn first example failed

### ISSUE THREAD

**Issue by iamadog3333** [automl/auto-sklearn#713]
Hi,
   I have tried the first auto-sklearn example.

```
import autosklearn.classification
import sklearn.model_selection
import sklearn.datasets
import sklearn.metrics

X, y = sklearn.datasets.load_digits(return_X_y=True)
X_train, X_test, y_train, y_test = \
        sklearn.model_selection.train_test_split(X, y, random_state=1)

automl = autosklearn.classification.AutoSklearnClassifier()
automl.fit(X_train, y_train)
y_hat = automl.predict(X_test)
print("Accuracy score", sklearn.metrics.accuracy_score(y_test, y_hat))
```

But after more than 2 hours run, not a model fitted.
All I got is this log:

```
[WARNING] [2019-08-02 00:05:06,795:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-02 00:05:08,798:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-02 00:05:10,802:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-02 00:05:12,807:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
```

>

**Comment by iamadog3333** [comment]
More log information:

```
/home/zhan/.local/lib/python3.6/site-packages/sklearn/ensemble/weight_boosting.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
/home/zhan/.local/lib/python3.6/site-packages/autosklearn/evaluation/train_evaluator.py:197: RuntimeWarning: Mean of empty slice
  Y_train_pred = np.nanmean(Y_train_pred_full, axis=0)
[WARNING] [2019-08-02 02:02:02,073:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-02 02:02:02,083:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
Segmentation fault (core dumped)
zhangqiang@ubuntu:~/work/automl$ [WARNING] [2019-08-02 02:02:04,088:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-02 02:02:06,093:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-02 02:02:08,099:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-02 02:02:10,104:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-02 02:02:12,109:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-02 02:02:14,113:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
...
```

**Comment by bowen0603** [comment]
I am having the same issue but not seg fault. I am using macOS and installed automl through pycharm. See the running log below:

/Users/bowenyu/Documents/Projects/ORES/venv/lib/python3.7/site-packages/sklearn/ensemble/weight_boosting.py:29: DeprecationWarning: numpy.core.umath_tests is an internal NumPy module and should not be imported. It will be removed in a future NumPy release.
  from numpy.core.umath_tests import inner1d
/Users/bowenyu/Documents/Projects/ORES/venv/lib/python3.7/site-packages/sklearn/feature_extraction/text.py:17: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Mapping, defaultdict
/Users/bowenyu/Documents/Projects/ORES/venv/lib/python3.7/site-packages/pyparsing.py:3168: FutureWarning: Possible set intersection at position 3
  self.re = re.compile(self.reString)
/Users/bowenyu/Documents/Projects/ORES/venv/lib/python3.7/site-packages/autosklearn/evaluation/train_evaluator.py:197: RuntimeWarning: Mean of empty slice
  Y_train_pred = np.nanmean(Y_train_pred_full, axis=0)
[WARNING] [2019-08-03 15:34:59,870:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!
[WARNING] [2019-08-03 15:34:59,882:EnsembleBuilder(1):d74860caaa557f473ce23908ff7ba369] No models better than random - using Dummy Score!

**Comment by applon** [comment]
I am having the same issue how to resolve

**Comment by soumya1704** [comment]
This seems to be similar to
https://github.com/automl/auto-sklearn/issues/688#issuecomment-515984588

Try setting up a conda env, do `conda install gxx_linux-64 gcc_linux-64 swig` and then `pip install auto-sklearn`. This time, on fitting, you'll still get the "Dummy Score" warnings, but they will stop appearing after about 20-30 mins. You'll get a tuned model in an hour

**Comment by mfeurer** [comment]
Thanks a lot @soumya1704 for giving an answer.

If the problem still exists, please reopen and post the logfile, too (the logfile is stored in the `tmp_folder`, which you can pass to the constructor).
